vm/src/stdlib/socket.rs

```diff
@@ -445,11 +445,11 @@ fn socket_inet_ntoa(packed_ip: PyBytesRef, vm: &VirtualMachine) -> PyResult {
     Ok(vm.new_str(Ipv4Addr::from(ip_num).to_string()))
 }
 
-fn socket_hton<U: num_traits::int::PrimInt>(host: U, _vm: &VirtualMachine) -> U {
+fn socket_hton<U: num_traits::int::PrimInt>(host: U) -> U {
     U::to_be(host)
 }
 
-fn socket_ntoh<U: num_traits::int::PrimInt>(network: U, _vm: &VirtualMachine) -> U {
+fn socket_ntoh<U: num_traits::int::PrimInt>(network: U) -> U {
     U::from_be(network)
 }
```

Comment:
I think we could replace these functions with just `new_function(u{16,32}::{to,from}_be)`; I'm not sure if that's a good idea but we're giving the type arguments in the `make_module` function anyway.